elastic/kibana
# Enable filters for the alert search bar on the Observability Alerts page

Fixes #158400
Resolves #177431

## Summary

This PR adds the filter option to the alert search bar.

https://github.com/elastic/kibana/assets/12370520/d476dec4-e540-4379-b2cf-076fd27c13a0


### 🧪 How to test
- Go to the Observability > Alerts page
- Use filter and saved query functionalities, they should work as expected.
- APM > Alerts tab should not show the filters, we can enable them separately if we want to.

Inspired by what @umbopepato implemented for the global alerts page!
